client/app/scripts/liveblog-bloglist/controllers/blog-list.js

```diff
@@ -489,4 +489,13 @@ export default function BlogListController(
     $scope.$on('$routeUpdate', fetchBlogs);
     // Reload bloglist on membership's approval
     $scope.$on('blogs', fetchBlogs);
+
+    // Check for bandwidth
+    api.get('/bandwidth/current').then((bandwidthUsage) => {
+        const percentageUsed = parseFloat(bandwidthUsage.percentageUsed.toFixed(1));
+
+        if (percentageUsed >= 90) {
+            notify.error(`Warning: You have used ${percentageUsed}% of your allocated bandwidth.`);
+        }
```

Comment:
I'd suggest improving this a little bit more. Like a modal dialog and show more information. Also telling the customer that we'll reach out to them soon. @aleksandarjelicic what do you think?

Reply:
@BrianMwangi21 Where is this meassage showing now?
Modal dialog can be limiting, you have to close it to continue working, and it's hard to see it again.

Reply:
@aleksandarjelicic it's showing a small notification for now 
![image](https://github.com/user-attachments/assets/fd4538c3-0bd6-46a4-9174-f48f59a27803)

I was thinking maybe we can have an alert like this one: 

![image](https://github.com/user-attachments/assets/edce7964-0a8c-4a32-8e92-d266889594a6)

What do you think ?

Reply:
@BrianMwangi21 second solution is good. You can put it in top black bar, or even above it.